Source/santactl/Commands/SNTCommandMetrics.m

```diff
@@ -0,0 +1,181 @@
+/// Copyright 2021 Google Inc. All rights reserved.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///    http://www.apache.org/licenses/LICENSE-2.0
+///
+///    Unless required by applicable law or agreed to in writing, software
+///    distributed under the License is distributed on an "AS IS" BASIS,
+///    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+///    See the License for the specific language governing permissions and
+///    limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import <MOLXPCConnection/MOLXPCConnection.h>
+
+#import "Source/common/SNTCommonEnums.h"
+#import "Source/common/SNTConfigurator.h"
+#import "Source/common/SNTMetricSet.h"
+#import "Source/common/SNTXPCControlInterface.h"
+#import "Source/santactl/SNTCommand.h"
+#import "Source/santactl/SNTCommandController.h"
+
+@interface SNTCommandMetrics : SNTCommand <SNTCommandProtocol>
+@end
+
+@implementation SNTCommandMetrics
+
+REGISTER_COMMAND_NAME(@"metrics")
+
++ (BOOL)requiresRoot {
+  return NO;
+}
+
++ (BOOL)requiresDaemonConn {
+  return YES;
+}
+
++ (NSString *)shortHelpText {
+  return @"Show Santa metric information.";
+}
+
++ (NSString *)longHelpText {
+  return (@"Provides metrics about Santa's operation while it's running.\n"
+          @"  Use --json to output in JSON format");
+}
+
+- (NSString *)stringFromMetricFormat:(SNTMetricFormatType)format {
+  switch (format) {
+    case SNTMetricFormatTypeMonarchJSON: return @"monarchjson";
+    case SNTMetricFormatTypeRawJSON: return @"rawjson";
+    default: return @"Unknown Metric Format";
+  }
+}
+
+- (NSDictionary *)normalize:(NSDictionary *)metrics {
```

Comment:
Don't we have a nearly identical method somewhere already?

Reply:
We do. I rewrote it here in case we wanted to have normalization change for human consumption and I'm not sure we'd hit the [rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)) yet.

Reply:
I've moved this function to `common/SNTMetricSet.m`